web/impact/impact/v1/views/reserve_office_hour_view.py

```diff
@@ -210,3 +220,39 @@ def _response(self):
             'header': self.header,
             'detail': self.detail,
             'timecard_info': self.timecard_info})
+
+    def get_calendar_data(self, counterpart_name):
+        title = self.OFFICE_HOUR_TITLE.format(counterpart_name)
+        office_hour = self.office_hour
+        if office_hour.location is None:
+            timezone = "UTC"
+            location = "MassChallenge"
+        else:
+            timezone = office_hour.location.timezone
+            location = office_hour.location
+        return Add2Cal(
+            start=office_hour.start_date_time.strftime(
+                ADD2CAL_DATE_FORMAT),
+            end=office_hour.end_date_time.strftime(
+                ADD2CAL_DATE_FORMAT),
+            title=title,
+            description=self._get_description(),
+            location=location,
+            timezone=timezone).as_dict()
+
+    def _get_description(self):
+        if self.office_hour.topics:
+            topics_block = "Topics: {topics}".format(
+                topics=self.office_hour.topics)
+
+        if self.office_hour.meeting_info:
+            info_block = "Meeting info: {info}".format(
+                info=self.office_hour.meeting_info)
+
+        description = """
+        {description}
```

Comment:
Omit description, it is not needed here per Hugh
(Topics contains the startup's topics for this particular session, description is just the mentor's blurb about themself)

Reply:
We as per AC-7816. We need more items in the description